django_measurement/forms.py

```diff
@@ -62,8 +62,8 @@ def __init__(self, measurement, max_value=None, min_value=None,
                     " Instead got type %s" % (measurement, str(type(bidimensional_separator)),)
                 unit_choices = tuple((
                     (
-                        '{0}__{1}'.format(primary, reference),
-                        '{0}{1}{2}'.format(
+                        u'{0}__{1}'.format(primary, reference),
```

Comment:
Please drop those changes. The trailing `u` was only really needed prior 3.3. No supported Python version requires this syntactic sugar.